src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -2206,6 +2237,30 @@ export default class PlotsTab extends React.Component<IPlotsTabProps,{}> {
         return ( (this.vertSelection.dataType === AlterationTypeConstants.GENERIC_ASSAY || this.horzSelection.dataType === AlterationTypeConstants.GENERIC_ASSAY)
                   && this.vertTreatmentOptions.length === 0);
     }
+    
+    @computed get showUtilitiesMenu() {
+        return (this.plotDataExistsForTwoAxes || this.waterfallPlotIsShown) && (this.mutationDataCanBeShown || this.cnaDataCanBeShown);
+    }
+    
+    @computed get sharedPlotTitle():string {
```

Comment:
I would like for the name to be more descriptive. can you explain more what the purpose of this title is?

Reply:
As Pim suggested, there should be a title to describe the currently selected gene, and this title will go along with the svg in the exported file when users want to download the plot. I named it `sharedPlotTtile` because it appears in both the scatter plot and the box plot.

Reply:
I think the name is too vague, `shared` doesn't really tell a reader what it is. how about `scatterAndBoxTitle`?